package.json

```diff
@@ -176,14 +176,12 @@
   "resolutions": {
     "async": "^3.2.2",
     "es5-ext": "0.10.53",
-    "micromatch": "^4.0.0",
     "readable-stream": "^4.0.0",
     "shell-quote": "^1.7.3"
   },
   "_justification": {
     "async": "Versions of async prior to 3.2.2 are vulnerable to prototype pollution",
     "es5-ext": "Packages after 0.10.54 and at the moment up until 0.10.59 contain a protest message. A policy prevents us from using packages with protestware, therefore downgrading to the latest release without the message.",
-    "micromatch": "Version 3.x.x depends on decode-uri-component 0.2.0, which has a DoS vulnerability",
```

Comment:
This and the `yarn.lock` change seem like a mistake?

Reply:
Bad merge, cleaning up